dts/bindings/mmc/nxp,imx-usdhc-slot.yaml

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2019, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: NXP i.MX USDHC slot
+version: 0.1
+
+description: >
+    NXP i.MX USDHC slot
+
+inherits:
+    !include nxp,imx-usdhc.yaml
```

Comment:
I understand nxp,imx-usdhc-slot.yaml describes hw pins, while nxp,imx-usdhc.yaml describes how the mcu handles sd signals. Why not have a single yaml for all pins, interrupts, or any other configuration.
I would rather have a yaml split between generic mmc/sd and particular nxp configurations. In your current pull i don't see anything that would be nxp particular.

Reply:
according to your suggestion, I use generic mmc.yaml for mmc/sdhc.